sdk/python/feast/constants.py

```diff
@@ -115,7 +116,15 @@ class AuthProvider(Enum):
     CONFIG_SERVING_ENABLE_SSL_KEY: "False",
     # Path to certificate(s) to secure connection to Feast Serving
     CONFIG_SERVING_SERVER_SSL_CERT_KEY: "",
-    # Default connection timeout to Feast Serving and Feast Core (in seconds)
+    # Default Feast Job Service URL
+    CONFIG_JOB_SERVICE_URL_KEY: "localhost:6568",
+    # Enable or disable TLS/SSL to Feast Job Service
+    CONFIG_JOB_SERVICE_ENABLE_SSL_KEY: "False",
+    # Path to certificate(s) to secure connection to Feast Job Service
+    CONFIG_JOB_SERVICE_SERVER_SSL_CERT_KEY: "",
+    # Enable or disable Feast Job Service
+    CONFIG_JOB_SERVICE_ENABLED: "False",
```

Comment:
Do we need this? Can we enable the JS if the url is provided?

Reply:
@tsotnet see my PR i had to implement the same thing

Reply:
Got it, merged